fastlane/lib/fastlane/runner.rb

```diff
@@ -149,6 +153,12 @@ def execute_action(method_sym, class_ref, arguments, custom_dir: '..')
       end
     end
 
+    def execute_flow_block(block, current_platform, lane, parameters)
+      # Call the platform specific block and then the general one
+      block[current_platform].call(lane, parameters) if block[current_platform] && current_platform
+      block[nil].call(lane, parameters) if block[nil]
```

Comment:
I don't think this will work.

The comment on 157, which was brought over from previous locations indicates that we should call the platform specific one, then the general one. An `if/elsif/else` would not allow this to happen.